core-feature-pack/src/main/resources/content/bin/jboss-cli.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
+. ./common.sh
+
```

Comment:
This needs to happen after the `$DIRNAME` is configured and should be `. "$DIRNAME/common.sh`. Without that it can't be launched from any other directory than bin, e.g. `cd $JBOSS_HOME/bin && ./jboss-cli.sh`.